app/src/main/java/com/sun/colornotetaking/ui/home/MainActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.sun.colornotetaking.ui.home;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+
+import com.sun.colornotetaking.R;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MainActivity extends AppCompatActivity {
+
+    private NavigationView mNavigationView;
+    private DrawerLayout mDrawerLayout;
+    private Toolbar mToolbar;
+
+    private Map<String, String> mTitleMap;
+    private static final String TAG_HOME = "home";
+    private static final String TAG_REMINDER = "reminder";
+    private static final String TAG_LABEL = "label";
+    private static final String TAG_RECYCLE_BIN = "recycler bin";
+    private static final String TAG_SETTING = "setting";
+    private static final String TAG_ABOUT = "about author";
+    private String mCurrentTag = TAG_HOME;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        initView();
+        initNavigationView();
+        initToolbarTitle();
+        if (savedInstanceState == null) {
+            mCurrentTag = TAG_HOME;
+            loadFragment();
+        }
+    }
+
+    private void initView() {
+        mToolbar = findViewById(R.id.tool_bar);
+        setSupportActionBar(mToolbar);
+        mToolbar.setNavigationIcon(R.drawable.ic_navigation_black_24dp);
+        mNavigationView = findViewById(R.id.navigation_view);
+        mDrawerLayout = findViewById(R.id.layout_drawer);
+    }
+
+    private void setToolbarTitle() {
+        getSupportActionBar().setTitle(mTitleMap.get(mCurrentTag));
+    }
+
+    private Fragment getFragment() {
```

Comment:
```suggestion
    private Fragment getFragment(String tag) {
```
viết ntn thì clear hơn